docs/wiki/ALZ-AMA-Migration-Guidance.md

```diff
@@ -0,0 +1,425 @@
+
+### Parity gaps
+
+Please check the most recent information on parity gaps before you begin:
+
+- [Known parity gaps for solutions that may impact your migration](https://learn.microsoft.com/en-us/azure/azure-monitor/agents/azure-monitor-agent-migration#known-parity-gaps-for-solutions-that-may-impact-your-migration)
+- [Microsoft Sentinel Gap analysis between agents](https://learn.microsoft.com/en-us/azure/sentinel/ama-migrate#gap-analysis-between-agents)
+- [Change Tracking and Inventory using Azure Monitoring Agent doesn't support or has the following limitations](https://learn.microsoft.com/en-us/azure/automation/change-tracking/overview-monitoring-agent?tabs=win-az-vm#current-limitations)
+
+### MDfC Defender for Servers
+
+All Defender for Servers features and capabilities will be provided through a single agent Microsoft Defender for Endpoint (MDE) integration, complemented by agentless capabilities, without dependency on either Log Analytics Agent (MMA) or Azure Monitoring Agent (AMA). Please refer to the following blog post containing the latest information [Microsoft Defender for Cloud - strategy and plan towards Log Analytics Agent (MMA) deprecation - Microsoft Community Hub](https://techcommunity.microsoft.com/t5/microsoft-defender-for-cloud/microsoft-defender-for-cloud-strategy-and-plan-towards-log/ba-p/3883341)
+
+### This guide covers the following topics
+
+- **Assess current state:** Identify and determine the steps required to migrate to AMA.
+- **Update Azure Landing Zones:** Guidance and automation to update your Azure Landing Zones components. Automation helps configure the following tasks:
+  - Deploy User Assigned Managed Identity
+  - Deploy Data Collection Rules
+  - Update Policy and Initiative definitions
+  - Remove Legacy Policy Assignments
+  - Remove Legacy Solutions
+  - Assigning new Policies and Initiatives
+  - Assign permissions to Landing Zones managed identity
+  - Policy remediation
+- **Removing MMA and additional steps:** Depending on the discoveries during the initial assessment you may need to run additional steps before you can remove MMA. Please refer to the tooling and guidance provided by the product team:
+  - Configure additional Data collection Rules (DCR Config Generator)
+  - AMA Migration for Microsoft Sentinel. Configuring connectors to send events via AMA
+  - Installing AMA on Non-Azure VMs with Azure Arc
+  - Update your Hybrid Workers (V1) to Extension based Hybrid Workers (V2)
+  - Custom settings for Change Tracking data types
+  - Migrating schedules to Azure Update Manager
+  - Removing MMA Agent
+
+### Migration paths
+
+1. [Migrate from MMA to AMA](#migrate-from-mma-to-ama). Applies to releases:
+   a. 2024-01-07 and earlier
+1. [Update to latest AMA release](#update-to-latest-ama-release). Applies to releases:
+   a. 2024-04-24
+   a. 2024-03-08
+   a. 2024-03-04
+   a. 2024-02-14
+   a. 2024-02-12
+   a. 2024-02-07
+   a. 2024-02-05
+   a. 2024-01-31
+
+## PowerShell script
```

Comment:
Should we add a bigger warning here saying do not use if an ALZ TF or Bicep customer?

Bicep wont be as bad to clear up but TF will be a mess and lots of imports

Reply:
I moved the powershell to a separate doc and added a warning.